scripts/esbuild.mjs

```diff
@@ -0,0 +1,18 @@
+import { posix } from 'path';
+import { globSync } from 'glob';
+import { build } from 'esbuild';
+
+const cwd = process.cwd();
+const dist = posix.join(cwd, 'dist');
+const pattern = posix.join(cwd, 'src/**/*.{js,ts}');
+const files = globSync(pattern);
+
+await build({
+  entryPoints: files,
+  bundle: process.argv.includes('--bundle'),
+  format: 'cjs',
+  outdir: dist,
+  platform: 'node',
+  target: 'es2021',
```

Comment:
Also, should the target read from `tsconfig.json` too?

Reply:
3f5f368b197226466f38ba3a237ad6745edf0156